panther_driver/src/hardware_node.py

```diff
@@ -202,6 +181,30 @@ def _handle_set_bool_srv(self, value: bool, pin: int, name: str) -> SetBoolRespo
     def _validate_gpio_pin(self, pin: int, value: bool) -> bool:
         return self._read_pin(pin) == value
 
+    @staticmethod
+    def _shutdown_host() -> None:
+        # EXAMPLE: https://www.linode.com/docs/guides/use-paramiko-python-to-ssh-into-a-server/
+        pkey = paramiko.RSAKey.from_private_key_file("/root/.ssh/id_rsa")
+        client = paramiko.SSHClient()
+        policy = paramiko.AutoAddPolicy()
+        client.set_missing_host_key_policy(policy)
+        client.connect(IP, username=USERNAME, pkey=pkey)
+        _stdin, _stdout, _stderr = client.exec_command("sudo shutdown now")
+        print(_stdout.read().decode())
```

Comment:
```suggestion
        _, stdout, _ = client.exec_command("sudo shutdown now")
        rospy.loginfo(f'[{rospy.get_name()}] {stdout.read().decode()}')
```

`_stdin` and `_stderr` are not used anywhere, so they can be removed. Also, `_` before variables that aren't class variables, but variables defined inside function is bad practice.

This is also a ROS node, so it has to use `rospy.loginfo` instead of `print`